js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -625,6 +625,17 @@ const DropDownEditor = TextBox.inherit({
     },
 
     _dimensionChanged: function() {
+        // This part hides the popup on window resize if an editor was hidden (T1133813)
```

Comment:
It's clear by code, no need to write comment
```suggestion
        // This part hides the popup on window resize if an editor was hidden (T1133813)
```

Reply:
https://github.com/DevExpress/DevExtreme/pull/23483/commits/709cfa030c245fe825d90854ffcbc9655bae3ca2